### DISCUSSION TITLE
Why are devices that are connected to front or rear ports no longer displayed in the connection column of the interfaces?

### DISCUSSION THREAD

**Discussion by julianstolp** [netbox-community/netbox#14819] [Q&A]
This was a good feature to see the connected devices on the switch interfaces, even if they were open front ports. As is often the case in office buildings, for example.
Is there a reason for this change? I dont see why this would be removed.

**Comment by candlerb** [comment]
There are two columns, "Link Peers" and "Connection": use the "Configure Table" button to make them visibile if they are not.

"Link Peers" shows where the cable immediately goes (e.g. to a front port). "Connection" shows the far-end interface, if there is a complete end-to-end path.

**Reply by julianstolp** [reply]
Yes I know, but why is a complete path, a "Connection" only interface to interface? A path could be completed by any cabled connection it should not matter if it is a front, rear or interface port. I would define the connection as connected device at the far end, doesn't matter which interface type.

![image](https://github.com/netbox-community/netbox/assets/41152397/e6c4cb46-1963-409e-9034-c12cc2bdd44c)
![image](https://github.com/netbox-community/netbox/assets/41152397/9a1714f0-1497-46b4-8358-bf0f44a9d76d)

**Reply by DanSheps** [reply]
That is not a complete path.

Lets say you have the following:

* A (24port Switch)
* B (24 Port Switch)
* C (24 Port Patch Panel)
* D (24 Port Patch Panel)

What you have is this:

```
[A:1] <> [C:FP1 -- C:RP1] <> [D:RP1 -- D:FP1]
```

A complete path would be:

```
[A:1] <> [C:FP1 -- C:RP1] <> [D:RP1 -- D:FP1] <> [B:1]
```

A complete path is a cable path that starts and ends on a cable endpoint (Interface, Power Port, Power Outlet, Console Port, Console Server Port, Circuit Termination, etc).

This is why you don't see anything under connection

**Reply by julianstolp** [reply]
Yes, I know the definition, but I don't understand why it is kept so narrow. In my opinion, every device at the far end, regardless of the interface type combination, is a connection. Should I discuss this in an FR?
